docs/dashboard/catalog/gift-cards.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Gift Cards
+---
+
+:::note
+Gift Cards are available in Saleor 3.1 (alpha stage). For production purposes, we recommend using the latest beta version of Saleor 3.0.
+:::
+
+This page contains information on how to manage gift cards by selecting a specific currency, sending them to customers, or setting expiring dates.
+
+## Introduction
+Gift cards are timeless and universal gifts. They are suitable for customers, employees, CEOs, business partners, or family members. Issuing gift cards can increase your sales and provide a better customer experience.
+
+## How to
+### How to create a gift card?
```

Comment:
```suggestion
### How to create a single gift card?
```

There's also a "bulk creation" feature that's worth mentioning.

Reply:
There is an additional section - "how to create multiple gift cards"